src/objects/RecurringDonationSchedule__c.object

```diff
@@ -153,6 +153,18 @@
     <enableSearch>false</enableSearch>
     <enableSharing>true</enableSharing>
     <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Active__c</fullName>
+        <description>Establishes whether the Recurring Donation Schedule is active or inactive. Read only</description>
```

Comment:
@Sam We decided to add this new formula field to centralize the logic for what an Active schedule record is. Would you mind reviewing the label and help text on these?

Reply:
@sam-knox Also add the new metadata to the PR info.

Reply:
@force2b - Gotcha! That's true. So End Date on the RD is always set by the related Schedule record then. They're always in sync.

Reply:
There is no end date on the RD record. The Period, Frequency, DayOfMonth and StartDate fields on the RD are what is used to create/update Schedule records. Schedule records aren't visible or editable, so they don't drive anything on the RD; other than the calculation for Next Donation Date and eventually CurrentYearExpectedValue.

Reply:
Got it!